### PR TITLE
fix() : ignore empty parameters of strategies in generated json files

### PR DESCRIPTION
**Pull request by RedouaeElalami**
ignore  the empty parameters in strategies while generating json files

### CODE REVIEW

**Review by iguissouma — APPROVED** [review]

File: src/test/resources/get-people-multiple-assertions-soft-strategy.chuntey.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (bessonm) [inline src/test/resources/get-people-multiple-assertions-soft-strategy.chuntey.json:1]: ! filename. "chu**nt**ey" -> chu**tn**ey

File: src/test/kotlin/com/chutneytesting/kotlin/dsl/ChutneyScenarioDslTest.kt [hunk 2]
```diff
@@ -111,4 +111,27 @@ class ChutneyScenarioDslTest {
 
     }
 
+    @Test
+    fun `abe to create chutney scenario using kotlin dsl with softAssertions`() {
```
Comment (bessonm) [inline src/test/kotlin/com/chutneytesting/kotlin/dsl/ChutneyScenarioDslTest.kt:115]: abe -> ab**l**e

File: src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyScenarioDsl.kt [hunk 3]
```diff
@@ -68,7 +68,7 @@ class ChutneyScenarioBuilder(val id: Int? = null, val title: String = "") {
     fun build(): ChutneyScenario = ChutneyScenario(id, title, description, givens, `when`, thens)
 
 }
```
Comment (bessonm) [inline src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyScenarioDsl.kt:70]: Add an empty line

**Review by RedouaeElalami — COMMENTED** [review]
done

**Review by bessonm — APPROVED** [review]

**Pull request merged**